### PR TITLE
Adding new sticker rendition and output

### PR DESCRIPTION
**Pull request by liamegan**
- Adding simplegl library
- Adding renderSticker helper
- Updating exportStickerbook to use renderSticker
- updating version #

### CODE REVIEW

File: src/helpers.js [hunk 1]
```diff
@@ -124,80 +125,17 @@ export async function exportStickerbook({
...
+    console.log('------------')
+    window.loadedStickers = loadedStickers;
+
     for (let i = 0; i < loadedStickers.length; i++) {
       const sticker = sortedStickers[i];
-      const stickerImage = loadedStickers[i];
-
-      // @marlonmarcello Thanks to Liam for all this magic bellow.
-      // It was just slightly adapted for this use case, but
-      // all the logic and math was him. Love you man.
```
Comment (marlonmarcello) [inline src/helpers.js:133]: ❤️

Comment (liamegan) [reply src/helpers.js:133]: <3

**Comment by andrewrubin** [comment]
Hey @liamegan thanks for all this!

I am having some issues when testing this locally. Getting a load of errors, I think related to a compiler step in one of these libraries. Have a look at the error logs attached. It could very well be the bundler on the stickerbook. Let me know your thoughts when you have time to look into it — maybe it's just a matter of switching some bundler settings to not obfuscate code (I have not looked into this yet). 

Screen Shot 2023-03-15 at 16 25 37

**Comment by liamegan** [comment]
This is a super weird one... Are you able to tell me whether this works if
you just drop the js files into a folder (you'll also need to import
wtc-simplegl)?

Thanks,
L

---------------------------------------------------------------
Liam Egan  |  We The Collective Design Inc.

37 Dunlevy Ave.
Vancouver BC V6A 3A3

Phone: +1 (604) 363-6466

www.wethecollective.com

*I respectfully acknowledge and am honoured to live, work and play within
the ancestral, traditional, and unceded territory of the Coast Salish
Peoples, specifically the territory of the xʷməθkʷəy̓əm (Musqueam),
Sḵwx̱wú7mesh (Squamish), and Sel̓íl̓witulh (Tsleil-Waututh) Nations.*

**Comment by andrewrubin** [comment]
> whether this works if you just drop the js files into a folder

It does work that way, yeah. We can solve that as necessary, all good.

Error on the front-end when exporting the stickerbook though 😢 
`Uncaught (in promise) TypeError: Failed to execute 'texImage2D' on 'WebGL2RenderingContext': Overload resolution failed.`

Let's dig into it tomorrow morning!

**Pull request merged**